src/modules/beam/gather/instructions/beam.gather.factories.ts

```diff
@@ -122,44 +129,50 @@ The final output should reflect a deep understanding of the user's preferences a
     addLabel: 'Add Breakdown',
     cardTitle: 'Evaluation Table',
     Icon: TableViewRoundedIcon,
-    description: 'Analyzes and compares AI responses, offering a structured framework to support your response choice.',
+    description: 'Analyzes and compares AI responses, offering a structured framework to support your response choice. Model names are hidden and coded (R1, R2, etc.) to remove potential bias.',
     createInstructions: () => [
       {
         type: 'chat-generate',
         label: 'Evaluation',
         method: 's-s0-h0-u0-aN-u',
         systemPrompt: `
-You are an advanced analytical tool designed to process and evaluate a set of AI-generated responses related to a user\'s query.
+You are an advanced analytical tool designed to process and evaluate a set of AI-generated responses related to a user's query.
+
+Your objective is to organize these responses to aid decision-making effectively. Begin by identifying key criteria for evaluating the responses, with a heavier weight on Accuracy and Pertinence. 
+In addition, select at least two more criteria that you find logically relevant, ensuring a minimum of 4 criteria in total for a thorough evaluation. 
+For user prompts seeking creative responses, more heavily weigh criteria such as "Originality" and "Creativity", while removing "Accuracy" as criteria option.
 
-Your objective is to organize these responses in a way that aids decision-making.
-You will first identify key criteria essential for evaluating the responses based on relevance, quality, and applicability.
+Next, analyze each response against these chosen criteria.
 
-Then, you will analyze each response against these criteria.
+Finally, synthesize your findings into a table, providing a clear overview of how each response measures up. Ensure to include Accuracy and Pertinence among your criteria (unless a creative query) and add any
+other criteria you find logically relevant, aiming for a total of at least 4 criteria.`.trim(),
 
-Finally, you will synthesize your findings into a table, providing a clear overview of how each response measures up. Start by identifying orthogonal criteria for evaluation (up to 2 for simple evaluations, up to 6 for many pages of input text).`.trim(),
         userPrompt: `
+
 Now that you have reviewed the {{N}} alternatives, proceed with the following steps:
 
-1. **Identify Criteria:** Define the most important orthogonal criteria for evaluating the responses. Identify up to 2 criteria for simple evaluations, or up to 6 for more complex evaluations. Ensure these criteria are distinct and relevant to the responses provided.
+1. **Identify Criteria:** Define the most logically relevant and essential orthogonal criteria for evaluating the responses. Always include Accuracy and Pertinence as primary criteria. 
```

Comment:
I spent a lot of time debating with AI itself over what really matters to get a net higher quality fusion response. Relevancy never quite fit, and I think pertinence nails it. Accuracy is tricky, as I still think the grading of accuracy is only discovered by apparent inconsistencies amongst the group, and the grading model doesn't know what it doesn't know, if you know what I mean. It may not recognize a different "correct" answer that it didn't already know, I think? As far as always including "accuracy" and "pertinence", I included some exceptions to account for edge cases (creative queries).

Reply:
Accuracy is tricky also because it can mean different things to different models. I'm almost leaning towards preferring Pertinence over accuracy.

Reply:
Yeah I need to think more. "correctness"? Here's some we can consider:

	•	Precision: Exactness in measurement or performance.
	•	Exactness: The degree of conformity to a standard or truth.
	•	Fidelity: Faithfulness to the original or to a standard.
	•	Authenticity: Genuineness or being true to one’s own character or the original.
	•	Veracity: Conformity to facts; accuracy.
	•	Truthfulness: Telling or inclined to tell the truth.
	•	Reliability: Consistency of measurement or performance over time.
	•	Validity: The extent to which a concept, conclusion, or measurement is well-founded and corresponds accurately to the real world.
	•	Credibility: Worthy of belief or confidence.
	•	Accuracy: Correctness or precision of information or measurement.
	•	Conformity: Agreement or compliance with standards, rules, or laws.
	•	Consistency: Uniformity or steadiness in quality or performance.
	•	Exactitude: The quality of being exact or precise in detail.
	•	Meticulousness: Extreme or excessive care in the consideration or treatment of details.
	•	Rigor: Strictness, severity, or thoroughness in maintaining standards.
	•	Faithfulness: Accuracy in reproducing a sound or image.
	•	Correctness: Free from error; in accordance with fact or truth.
	•	Verisimilitude: The appearance of being true or real.
	•	Adherence: The quality of sticking strictly to standards, rules, or practices.
	•	Exactness: The quality of being very accurate and precise.
	•	Precision: Exactness in the language model’s responses to queries.
	•	Fidelity: Faithfulness of the model’s output to the facts or source material.
	•	Authenticity: Genuineness of the responses, reflecting true understanding.
	•	Veracity: Adherence to truth and accuracy of factual information provided.
	•	Relevance: The degree to which the model’s responses pertain to the given tasks or questions.
	•	Adaptability: The model’s ability to adjust its responses based on new information or feedback.
	•	Critical Thinking: The model’s ability to analyze, evaluate, and synthesize information in its responses.
	•	Innovation: Originality and creativity in generating solutions or responses.
	•	Ethical Judgment: The model’s demonstration of ethical considerations and avoidance of harm in its responses.
	•	Factual Accuracy: Correctness of factual statements, essential for trustworthiness.
	•	Logical Reasoning: Clear and sound reasoning in constructing arguments or explanations.
	•	Problem-Solving Skills: Effectiveness in identifying and proposing solutions.
	•	Detail Orientation: Attention to and incorporation of significant details in responses.
	•	Coherence: Logical consistency and clarity in responses, ensuring they are understandable and follow a logical flow.
	•	User Engagement: The ability to maintain the user’s interest and promote further interaction through engaging and relevant content.
•	Comprehensiveness: The extent to which the model can cover all relevant topics or knowledge areas for a given task.
	•	Synthesis Ability: The model’s capacity to integrate and combine information from various sources into a coherent whole, showing depth of understanding.
	•	Clarity: The ease with which users can understand the model’s responses, emphasizing clear and accessible language.
	•	Insightfulness: The depth of understanding and the ability to provide novel insights or perspectives in responses.
	•	Responsiveness: The precision with which the model addresses and adapts to the specific parts of a prompt or question, including the nuanced understanding of user intent.
	•	Emotional Intelligence: The model’s ability to recognize and appropriately respond to emotional cues in text, demonstrating sensitivity to the emotional context.
	•	Cultural Competence: Awareness of and sensitivity to cultural differences, ensuring responses are appropriate and respectful across diverse contexts.
	•	Technical Proficiency: The accuracy and depth of knowledge in responses to queries requiring specialized understanding or technical expertise.
	•	Verisimilitude: The appearance of realism or believability in the model’s outputs, especially in creative or generative tasks.